src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -63,6 +63,14 @@ public static class CodeActions
 
             public const string AddUsing = "AddUsing";
 
+            /// <summary>
+            /// Remaps without formatting the resolved code action edit
+            /// </summary>
+            public const string Remap = "Remap";
```

Comment:
Hmmm I feel like `Remap` seems too normal in the sense that people wont be scared by it (and they probably should be). Lets update the name to reflect the risk. `RemapRaw`, `UnformattedRemap` or something else. I'm bad at naming 😆